ui/src/publication/table/publication-table-row.tsx

```diff
@@ -7,14 +7,43 @@ import { Operation } from 'publication/publication-model';
 export type PublicationTableRowProps = {
     name: string;
     trackNumbers: TrackNumber[];
-    changedKmNumbers: KmNumber[];
+    changedKmNumbers?: KmNumber[];
     operation: Operation;
     publicationTime: TimeStamp;
     publicationUser: string;
     definition: string;
     ratkoPushTime: TimeStamp | null;
 };
 
+const formatKmNumber = (kmNumbers: KmNumber[]) => {
+    return kmNumbers
+        .sort()
+        .reduce((acc, value) => {
+            if (acc.length == 0) acc.push([value]);
+            else {
+                const previousArr = acc[acc.length - 1];
+                const previousValue = previousArr[previousArr.length - 1];
+                const currentIntValue = parseInt(value);
+                const previousIntValue = parseInt(previousValue);
+
+                //Group kilometer numbers with extensions
+                //For example, [0001, 0001A, 0001B, 0003] => 0001 - 0001B, 0003
```

Comment:
Tämä on ehkä vähän tarpeetonta kun ainakaan toistaiseksi meillä ei ole noita kirjain-liitteellisiä kilometrejä. Ei sinällään haittaakaan.

Reply:
Mietin että kannattaisiko nämä välit laskea jo bäkkärillä. Nythän periaatteessa jos meillä on 2 lähekkäistä, mutta erillistä muutosta esim. ratakilometrin 0001 lopusta 0002 alkuun ja 0003 lopusta jonnekin eteenpäin, niin nyt tuo taitaisi näkyä yhtenä, ratakilometriltä 0001 alkavana välinä. Tämä voi olla tosin myös haluttua näin ja riippuu siitä mikä operaattoreita oikeasti kiinnostaa tässä columnissa, mutta oma logiikka sanoisi että noissa tilanteissa nuo voisi näyttää erillisinä väleinä jos/kun kyseessä ei ole sama muutos.

Reply:
Hmm.. hyvä huomio. 
Ajattelisin, että tässä näkymässä ratkaisevana tekijänä on julkaisuhetki eli jos muutokset on julkaistu samaan aikaan --> "yksi muutos".